### ISSUE TITLE
Add support for BLAKE3 assembly statements.

### ISSUE THREAD

**Issue by fengjian** [AluVM/aluasm#17]

**Comment by dr-orlovsky** [comment]
Yes. And we actually need to add it to assembly parser with rust macros in the original aluvm repo

**Comment by fengjian** [comment]
I have added BLAKE3 assembly statements, but it requires updating aluvm dependency. We need to add a new beta tag.

**Comment by dr-orlovsky** [comment]
You can just use `[patch.crates-io]` section for now
